ckanext/canada/tables/ati.yaml

```diff
@@ -52,6 +50,24 @@ resources:
     datastore_type: int
```

Comment:
I expect this will cause problems when importing the data. Have you tried importing the existing data? We likely need to clean up the old data so it validates as `int`s here.

Reply:
(that's something to add to the migration script)